diesel_cli/src/print_schema.rs

```diff
@@ -697,20 +697,11 @@ impl<'a> Display for ColumnDefinitions<'a> {
                 }
 
                 // Write out attributes
-                if column.rust_name != column.sql_name || column.ty.max_length.is_some() {
-                    let mut is_first = true;
-                    write!(out, r#"#["#)?;
-                    if column.rust_name != column.sql_name {
-                        write!(out, r#"sql_name = {:?}"#, column.sql_name)?;
-                        is_first = false;
-                    }
-                    if let Some(max_length) = column.ty.max_length {
-                        if !is_first {
-                            write!(out, ", ")?;
-                        }
-                        write!(out, "max_length = {}", max_length)?;
-                    }
-                    writeln!(out, r#"]"#)?;
+                if column.rust_name != column.sql_name {
+                    writeln!(out, r#"#[sql_name = "{}"]"#, column.sql_name)?;
+                }
+                if let Some(max_length) = column.ty.max_length {
```

Comment:
@Ten0 this intends to fix an issue I ran into from #3552, where multiple comma-separated annotations would cause compilation errors -- github actions tests pass, although I am having some trouble testing it locally